GitUI/Editor/FileViewer.cs

```diff
@@ -1618,5 +1619,10 @@ public bool ShowSyntaxHighlightingInDiff
             public ToolStripButton IgnoreAllWhitespacesButton => _fileViewer.ignoreAllWhitespaces;
             public ToolStripMenuItem IgnoreAllWhitespacesMenuItem => _fileViewer.ignoreAllWhitespaceChangesToolStripMenuItem;
         }
+
+        public void SetGutterAvatars(List<GitBlameDisplay> gitBlameDisplays)
```

Comment:
The method is called "set avatars", yet its parameter isn't a collection of images. So the name is misleading.

Maybe something like this?
```suggestion
        public void SetGitBlameGutter(List<GitBlameEntry> blameEntries)
```